src/pages/attribute-dictionary.js

```diff
@@ -76,31 +80,34 @@ const AttributeDictionary = ({ data, pageContext }) => {
           }
         `}
       >
-        <PageTitle>New Relic data dictionary</PageTitle>
+        <PageTitle>{t('dataDictionary.title')}</PageTitle>
         <div
           css={css`
             grid-area: 'page-description';
           `}
         >
-          <p
-            css={css`
-              color: var(--secondary-text-color);
-              font-size: 1.125rem;
-            `}
-          >
-            This data dictionary lists and defines the{' '}
-            <Link to="/docs/using-new-relic/welcome-new-relic/getting-started/glossary#attribute">
-              attributes
-            </Link>{' '}
-            attached to New Relic events and other data objects (like Metric and
-            Span data).
-          </p>
-          <Callout variant={Callout.VARIANT.TIP}>
-            This dictionary does not contain data reported by Infrastructure
-            integrations. To learn about that data, see the{' '}
-            <Link to="/docs/integrations">integration documentation</Link>.
-          </Callout>
-
+          <Trans i18nKey="dataDictionary.intro" parent="div">
```

Comment:
It looks like we didn't have this `div` wrapping the content before, just the `p`. Could you instead merge this `Trans` component with its `p` child?

```js
<Trans parent='p' css={css`...`}>
  // ...
</Trans>
```

Reply:
oh right i can just add the emotion `css` prop on the component. sweet! done